pkgs/development/python-modules/jsonsir/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, setuptools
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "jsonsir";
+  version = "0.0.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "JsonSir";
+    hash = "sha256-QBRHxekx94h4Uc6b8kB/401aqwsUZ7sku787dg5b0/s=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  pythonImportsCheck = [ "jsonsir" ];
+
+  # No tests in PyPi release
+  doCheck = false;
```

Comment:
There are tests in the GitHub repository, however it is not tagged so it seems the PyPi release is more reliable in this case.